cinema/models.py

```diff
@@ -35,12 +38,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, extension = os.path.splitext(filename)
+    return os.path.join(
+        "uploads/movies/",
+        f"{slugify(instance.title)}-{uuid.uuid4()}.{extension}"
```

Comment:
Is really dot needed here?
```suggestion
        f"{slugify(instance.title)}-{uuid.uuid4()}{extension}"
```